src/test/resources/network/brightspots/rcv/test_data/test_set_1_exhaust_at_overvote/test_set_1_exhaust_at_overvote_cvr.json

```diff
@@ -799,45 +799,44 @@
   "Election" : [ {
     "@id" : "election-001",
     "@type" : "CVR.Election",
+    "Candidate" : [
+      {
+        "@id" : "candidate-a",
+        "Name" : "Candidate A Name"
+      },
+      {
+        "@id" : "candidate-b",
+        "Name" : "Candidate B Name"
+      },
+      {
+        "@id" : "candidate-c",
+        "Name" : "Candidate C Name"
+      },
+      {
+        "@id" : "candidate-d",
+        "Name" : "Candidate D Name"
+      }
+    ],
     "Contest" : [ {
       "@id" : "contest-001",
-      "@type" : "CVR.CandidateContest",
+      "@type" : "CandidateContest",
+      "Name" : "test set 1",
       "ContestSelection" : [ {
         "@id" : "cs-a",
-        "@type" : "CVR.ContestSelection",
-        "Code" : [ {
-          "@type" : "CVR.Code",
-          "OtherType" : "vendor-label",
-          "Type" : "other",
-          "Value" : "A"
-        } ]
+        "@type" : "CandidateSelection",
+        "CandidateIds" : [ "candidate-a" ]
```

Comment:
@tarheel Here we replace a 'Code' object with a link to a Candidate object.  Note that confusingly it is actually an array, and is correspondingly named CandidateIds.  This is to support "party ticket" options, and is a rare thing, but we all have to pay the price.  It's important to not the _multiplicity_ of links in the CDF UML spec.